src/tree/ResourceGroupTreeItem.ts

```diff
@@ -64,13 +65,8 @@ export class ResourceGroupTreeItem extends GroupTreeItemBase {
     }
 
     public async getNumOfResources(context: IActionContext): Promise<number> {
```

Comment:
Since the deletion is the only place to use this, we should move this code there, to ensure it's scoped to _only_ that scenario. We wouldn't want live updates coming from the web all the time.